seqr/views/utils/individual_utils.py

```diff
@@ -195,6 +193,27 @@ def delete_individuals(project, individual_guids, user):
     return families_with_deleted_individuals
 
 
+def _validate_delete_individuals(individuals_to_delete, error_type, query):
+    errors = []
+    invalid_individuals = individuals_to_delete.filter(**query).distinct().values_list('individual_id', flat=True)
+    if invalid_individuals:
+        errors.append(f'Unable to delete individuals with active {error_type}: {", ".join(sorted(invalid_individuals))}')
+    return errors
+
+
+def _validate_no_submissions(individuals_to_delete):
+    return _validate_delete_individuals(
+        individuals_to_delete, 'MME submission',
+        dict(matchmakersubmission__isnull=False, matchmakersubmission__deleted_date__isnull=True)
+    )
+
+
+def _validate_no_search_samples(individuals_to_delete):
```

Comment:
would it be clearer to name this function `_validate_no_search_and_no_submissions_samples`, 
just so it's clear in [line 177] above that the hail backend isn't just validating search, but both search and submissions?